awd/crossref.py

```diff
@@ -74,6 +74,19 @@ def create_dspace_metadata_from_dict(value_dict, metadata_mapping_path):
         return {"metadata": metadata}
 
 
+def is_valid_dspace_metadata(dspace_metadata):
```

Comment:
So, this seems to be checking if the corresponding value for a key is not nil. Does this account for the misspelled case that was seen earlier?

Reply:
Added the approved metadata fields to avoid that issue in the future